lib/capistrano/tasks/sidekiq.cap

```diff
@@ -120,6 +120,11 @@ namespace :sidekiq do
     end
   end
 
+  def queue_for_host(host)
+    sidekiq_roles = host.roles & fetch(:sidekiq_role)
+    fetch(:"#{ sidekiq_roles.first }_queue") || fetch(:sidekiq_queue)
```

Comment:
This is the only thing I don't like here, it's possible you could have a host with more than one sidekiq role, in which case I'm just arbitrarily selecting the first one... Do you think that should raise an error instead?